frontend/src/components/WildcardSelect.vue

```diff
@@ -86,76 +97,62 @@ export default {
   },
   data () {
     return {
-      wildcardVariablePart: undefined,
+      wildcardVariablePartPrefix: undefined,
+      wildcardVariablePartSuffix: undefined,
       wildcardSelectedValue: {},
       valid: undefined
     }
   },
   computed: {
     validationErrors () {
       return {
-        wildcardVariablePart: {
-          required: `${this.wildcardTextFieldLabel} is required`
-        },
         wildcardSelectedValue: {
-          required: `${this.wildcardSelectLabel} is required`
+          required: `${this.wildcardSelectLabel} is required`,
+          prefixRequired: 'Prefix is required',
+          suffixRequired: 'Suffix is required'
         }
       }
     },
-    textFieldClass () {
-      return this.wildcardSelectedValue.startsWithWildcard ? 'textFieldStartClass' : 'textFieldEndClass'
-    },
-    selectClass () {
-      if (this.wildcardSelectedValue.customWildcard) {
-        return 'selectSmallClass'
-      }
-      return undefined
-    },
     wildcardSelectItemObjects () {
       return wildcardObjectsFromStrings(this.wildcardSelectItems)
     },
-    wildcardTextFieldLabel () {
-      if (this.wildcardSelectedValue.startsWithWildcard) {
-        return 'Prefix'
-      }
-      if (this.wildcardSelectedValue.endsWithWildcard) {
-        return 'Suffix'
-      }
-      if (this.wildcardSelectedValue.customWildcard) {
-        return `Custom ${this.wildcardSelectLabel}`
-      }
-      return undefined
-    },
     wildcardSelectHint () {
-      if (!this.wildcardSelectedValue.isWildcard) {
-        return undefined
-      }
-      if (this.wildcardSelectedValue.customWildcard) {
-        return `Specify custom ${this.wildcardSelectLabel}`
-      } else {
-        const label = this.wildcardSelectedValue.startsWithWildcard ? 'prefix' : 'suffix'
-        return `Selected wildcard value requires a ${label} which needs to be specified`
-      }
+      return `Floating Pool Name: ${this.internalValue}`
```

Comment:
```suggestion
      return `Custom Name: ${this.internalValue}`
```
maybe?

Reply:
yes. I just noticed hard coding it to Floating IP is a bad idea anyway as this component is kind of generic even if we currently only use it for floating IP. I'll double check this.